packages/ogmios/src/CardanoNode/OgmiosObservableCardanoNode/createObservableInteractionContext.ts

```diff
@@ -27,7 +27,7 @@ export interface InteractionContextProps {
    * 'OneTime' will close the connectino (and complete the interaction context observable)
    * after the 1st usage of emitted interaction context.
    */
-  interactionType: InteractionType;
+  interactionType: InteractionType; // REVIEW: Deprecate this? InteractionType is not used anymore in ogmios
```

Comment:
Yes, but probably do it as a separate commit as a cleanup step. So all interactions are "long running"? I'm guessing the availability of a HTTP API for "one-time" requests is the reason for the deprecation?

Reply:
Ogmios.createInteractionContext no longer expects an `interactionType`.

Reply:
Done here: [6ab0ec8](https://github.com/input-output-hk/cardano-js-sdk/pull/927/commits/6ab0ec8f536b3d72f472eee75002d2771f62c30f)